pages/common/grep.md

```diff
@@ -11,11 +11,15 @@
 
 `grep -r {{something}} .`
 
+- perform a case insensitive search 
+
+`grep -r {{match}}
```

Comment:
According to man, `-r` stands for recursive, not case insensitive search. See previous code example.